examples/hadoop/src/main/java/com/hazelcast/jet/examples/hadoop/cloud/avro/AmazonS3Avro.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.examples.hadoop.cloud.avro;
+
+import com.hazelcast.jet.examples.hadoop.HadoopAvro;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+
+/**
+ * A simple example adapted to read from and write to Amazon S3 object storage
+ * using HDFS source and sink. The example uses Apache Avro input and output
+ * format.
+ * <p>
+ * The job reads records from the given bucket {@link #BUCKET_NAME}, filters
+ * according to a field of the record and writes back the records to a folder
+ * inside that bucket.
+ * <p>
+ * To be able to read from and write to Amazon S3 object storage, HDFS needs
+ * couple of dependencies and the credentials for S3. Necessary dependencies:
+ * <ul>
+ *     <li>hadoop-aws</li>
+ *     <li>hadoop-client</li>
+ * </ul>
+ *
+ * @see <a href="https://hadoop.apache.org/docs/r3.0.3/hadoop-aws/tools/hadoop-aws/index.html">
+ * Hadoop-AWS Module</a> for more information
+ */
+public class AmazonS3Avro {
```

Comment:
I think we can delete all the cloud + avro examples. They don't add anything new since it's just the path that's different.